app/[components]/[category]/[collection]/page.jsx

```diff
@@ -46,7 +47,7 @@ async function getCollection(params) {
     `${params.category}-${params.collection}.mdx`
   )
 
-  const postItem = await fs.readFile(componentPath, 'utf-8')
+  const postItem = await fs.readFile(componentPath, 'utf-8').catch(() => notFound())
```

Comment:
I'm not a fan of `.catch()`.

Reply:
Ah, any other way you want Todo?

Reply:
```js
if (!postItem) {
  notFound()
}
```

Reply:
Already tried that, **it throws error, if I would done this**, results in 500 in production, error log in development